package/kedro_viz/launchers/cli.py

```diff
@@ -128,8 +128,14 @@ def run(
     params,
 ):
     """Launch local Kedro Viz instance"""
+    from kedro.framework.startup import bootstrap_project
+
     from kedro_viz.server import run_server
 
+    project_path = Path.cwd()
+
+    bootstrap_project(project_path)
```

Comment:
This is fine but instead of showing an error stack trace, it would be nice to show a custom error message with the exception, something like -

```python    
# check if the project_path points to a valid Kedro Project
    try:
        bootstrap_project(project_path)
    except Exception as exc:
        display_cli_message(f"ERROR: Unable to start Kedro-Viz : {exc} ", "red")
        return
 ```